src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `obj`. It will decrease code readability. Choose a more descriptive name that reflects the purpose of the variable, such as `styleObject`.